src/Store.js

```diff
@@ -216,7 +216,12 @@ class Store {
    * @param {Object} options
    */
   findAll = (type, options = {}) => {
-    const { fromServer, queryParams } = options
+    const { fromServer, queryParams, lazyLoad } = options
+
+    if (lazyLoad) {
```

Comment:
Should we throw an invariant error if they have set fromServer and lazy load together?

Reply:
This is gone now since we moved to having a separate method.